config/env_sample.json

```diff
@@ -0,0 +1,127 @@
+{
+    "/*  Required to set this with some value": "*/",
+    "/*  Run the code here to generate https://raw.githubusercontent.com/openwisp/ansible-openwisp2/master/files/generate_django_secret_key.py": "*/",
+    "DJANGO_SECRET_KEY": "<Generate Secret Key>",
+    "/*  Set the Timezone for the container (must rebuild if changed)": "*/",
+    "TZ": "America/Detroit",
+    "/*  CSV List of hosts allowed, no spaces in between commas": "*/",
+    "ALLOWED_HOSTS": [
+        "127.0.0.1",
+        "localhost"
+    ],
+    "/*  ID for your Google Analytics, defaults to nothing/disabled": "*/",
+    "GA_ID": "",
+    "/*  Any logs that django outputs, turn these off for production": "*/",
+    "DJANGO_LOG_LEVEL": "DEBUG",
+    "DJANGO_DEBUG": "true",
+    "DJANGO_TEMPLATE_DEBUG": "true",
+    "/*  Any other logs": "*/",
+    "ROOT_LOG_LEVEL": "INFO",
+    "/*  How long to cache some URL's on the client (Defualt 3600 seconds)": "*/",
+    "/*  CLIENT_CACHE_TIME=3600": "*/",
+    "/*  MySQL Configuration": "*/",
+    "/*  Mysql Django Engine name": "*/",
+    "MYSQL_ENGINE": "django.db.backends.mysql",
+    "/*  Local database name": "*/",
+    "MYSQL_DATABASE": "student_dashboard",
+    "/*  Local database user": "*/",
+    "MYSQL_USER": "student_dashboard_user",
+    "/*  Local database password": "*/",
+    "MYSQL_PASSWORD": "student_dashboard_pw",
+    "/*  Local database host": "*/",
+    "MYSQL_HOST": "student_dashboard_mysql",
+    "/*  Local database port": "*/",
+    "MYSQL_PORT": 3306,
+    "/*  Local database root password (optional)": "*/",
+    "MYSQL_ROOT_PASSWORD": "student_dashboard_root_pw",
+    "/*  Default Canvas Data id increment for course id, user id, etc": "*/",
+    "CANVAS_DATA_ID_INCREMENT": 17700000000000000,
+    "/*  Canvas Configuration": "*/",
+    "CANVAS_USER": "",
+    "/*  strings for construct file download url": "*/",
+    "CANVAS_FILE_PREFIX": "https://example.instructure.com/files/",
+    "CANVAS_FILE_POSTFIX": "/download?download_frd=1",
+    "/*  Paths to special login credentials (uses SAML)": "*/",
+    "GOOGLE_APPLICATION_CREDENTIALS": "/secrets/bq_cred.json",
+    "/*  ed_app field to look for in Big Query store": "*/",
+    "BIG_QUERY_ED_APP": "http://umich.instructure.com/",
+    "/*  LTI Configuration": "*/",
+    "/*  LTI is disabled by default": "*/",
+    "STUDENT_DASHBOARD_LTI": "False",
+    "/*  list LTI consumers as per https://github.com/rohitjose/django-lti-authusage": "*/",
+    "/*  ex: PYLTI_CONFIG_CONSUMERS={ \"LTI_CONSUMER_KEY_1\": { \"secret\": \"LTI_CONSUMER_SECRET_1\" } }": "*/",
```

Comment:
```
"/*  ex: PYLTI_CONFIG_CONSUMERS={ \"LTI_CONSUMER_KEY_1\": { \"secret\": \"LTI_CONSUMER_SECRET_1\" } }": "*/",
```
to
```
"/*  ex: "PYLTI_CONFIG_CONSUMERS": { "LTI_CONSUMER_KEY_1": { "secret": "LTI_CONSUMER_SECRET_1" } }": "*/",
```

Reply:
Thanks @andrew-gardener , I fixed this up.